file/prefetch_test.cc

```diff
@@ -275,8 +275,8 @@ TEST_P(PrefetchTest, ConfigureAutoMaxReadaheadSize) {
           break;
         case 1:
           // max_auto_readahead_size is set less than
-          // BlockBasedTable::kInitAutoReadaheadSize. So readahead_size remains
-          // equal to max_auto_readahead_size.
+          // initial_auto_readahead_size. So readahead_size remains equal to
```

Comment:
@mrambacher this test covers that. In case 0, max_auto_readahead_size = 0 and initial is default (8KB).

Reply:
@mrambacher  It also covers that initial_auto_readahead_size > max_auto_readahead_size > 0 in case 1.